packages/blocks/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cx from 'classnames';
+
+type RadioCardProps = {
+  startIcon?: React.ReactNode;
+  isSelected: boolean;
+  label: string;
+};
+
+export const RadioCard: React.FC<RadioCardProps> = ({
+  startIcon,
+  isSelected,
+  label,
+}) => {
+  return (
+    <div className={cx('t-radio-card', {'t-tag-blue': isSelected})}>
+      {startIcon && startIcon}
+      {label && label}
+    </div>
+  );
+};
```

Comment:
We should swap the hover states here for `isSelected`. If `true`, then _no_ hover state and _no_ pointer cursor (item is already selected so can't be chosen again) vs if `false` then there should be a hover state, pointer cursor and hover state.

This is the context it will be used in:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/57259/163288870-cb620fe5-ce39-412d-8d36-82ebd6ac07d1.png">